meeg_tools/utils/epochs.py

```diff
@@ -131,31 +131,63 @@ def create_metadata(epochs: Epochs):
         )
 
         # we can add boundaries of epochs
-        epoch_boundaries = [211, 212, 213, 214, 215, 216]
+        epoch_boundaries = [210, 211, 212, 213, 214, 215, 216]
+        practice_end_trigger = epoch_boundaries[0]
 
         edges = np.where(np.isin(epochs.events[..., 2], epoch_boundaries))[0]
 
         boundaries = dict(zip(epoch_boundaries, edges))
         logger.info("Found these indices for these epoch boundary events: ")
         logger.info("\n".join("{}\t{}".format(k, v) for k, v in boundaries.items()))
         for epoch_ind, epoch in enumerate(np.split(epochs.events, edges, axis=0)):
-            if epoch_ind != len(edges):
-                metadata.loc[
-                    metadata["time_in_samples"].isin(epoch[..., 0]), "epoch"
-                ] = (epoch_ind + 1)
+            if practice_end_trigger in boundaries:
+                epoch_number = epoch_ind
+            else:
+                epoch_number = epoch_ind + 1
+            metadata.loc[
+                metadata["time_in_samples"].isin(epoch[..., 0]), "epoch"
+            ] = epoch_number
 
         metadata.loc[
-            metadata["id"].isin([44, 45, 46, 47, 49, 144, 145, 146, 147, 149]), "answer"
+            metadata["id"].isin(
+                [44, 45, 46, 47, 49, 67, 68, 69, 70, 78, 144, 145, 146, 147, 149]
+            ),
+            "answer",
         ] = "incorrect"
         metadata.loc[
-            ~metadata["id"].isin([44, 45, 46, 47, 49, 144, 145, 146, 147, 149]),
+            ~metadata["id"].isin(
+                [44, 45, 46, 47, 49, 63, 64, 65, 66, 144, 145, 146, 147, 149]
+            ),
```

Comment:
This part will mark correct answers as the triggers that are **not** in that list, the logic was that answers are correct if they are not in the "incorrect" list of items above.

@vekteo